t/08-autotest.t

```diff
@@ -468,6 +468,26 @@ subtest 'test skipping tests' => sub {
     stderr_like { autotest::runalltests } qr/skipping/, 'Skipping Test Run';
 };
 
+subtest 'start_process' => sub {
+    my $fh;
+    open($fh, '<', \"fake-file");    # the file handle will be closed automatically inside start_process
```

Comment:
```suggestion
    open $fh, '<', \"fake-file";
```
The fact that variables go out of scope at the end of the block is something that we shouldn't have to comment everywhere.